modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -375,7 +381,7 @@ public void postProcessParameter(CodegenParameter parameter) {
         List<Map<String, Object>> imports = (List<Map<String, Object>>) operations.get("imports");
         for (Map<String, Object> im : imports) {
             im.put("filename", im.get("import"));
-            im.put("classname", getModelnameFromModelFilename(im.get("filename").toString()));
+            im.put("classname", im.get("classname")/*getModelnameFromModelFilename(im.get("filename").toString())*/);
```

Comment:
Minor suggestion: remove commented code: `/*getModelnameFromModelFilename(im.get("filename").toString())*/`

Reply:
Ok no problem but I'm not sure which branch to make the change ... could you point me?

Reply:
You should do it in `resolve_issue727` (the branch of this PR)

Reply:
Ok done.

Reply:
PR merged into master. Thanks for your contribution.